src/Transports.Subscriptions.Abstractions/MessageHandlingContext.cs

```diff
@@ -1,10 +1,11 @@
 ﻿using System;
 using System.Collections.Concurrent;
+using System.Collections.Generic;
 using System.Threading.Tasks;
 
 namespace GraphQL.Server.Transports.Subscriptions.Abstractions
 {
-    public class MessageHandlingContext : IDisposable
+    public class MessageHandlingContext : Dictionary<string, object>, IDisposable
```

Comment:
Not sure about that. Inside there is already a `ConcurrentDictionary` for some properties. Not yet thoroughly understood this repository.